sunpy/map/mapbase.py

```diff
@@ -1296,21 +1292,15 @@ def _mpl_imshow_kwargs(self, axes, cmap):
         """
```

Comment:
As @ehsteve and discussed in IRC, this should be removed and replaced by the plot_settings dict.